src/components/popover/_popover.scss

```
@@ -38,7 +38,6 @@
   backface-visibility: hidden;
   pointer-events: none;
   opacity: 0; /* 2 */
-  visibility: hidden; /* 2 */
   transition: /* 2 */
     opacity $euiAnimSlightBounce $euiAnimSpeedSlow,
     visibility $euiAnimSlightBounce $euiAnimSpeedSlow;
```

This needs to be removed as well, otherwise it still counts for browser detection
```suggestion
    opacity $euiAnimSlightBounce $euiAnimSpeedSlow;
```

+ same for the transition on line 50 below